lib/registrant.rb

```diff
@@ -0,0 +1,35 @@
+require "./spec/spec_helper"
+
+# The Registrant class represents a single person and their license data.
+# This includes their name, age, and whether they have a driving permit,
+# driver's license, and whether their license has been renewed.
+# Syntax:
+#
+# registrant = Registrant.new({name: "Bill", age: 18})
+#
+# Methods:
+#
+# registrant.earn_permit
+# registrant.permit?
+class Registrant
+  attr_reader :name, :age, :permit
+  attr_accessor :license_data
```

Comment:
Moving forward, see if you can avoid using attr_accessors :) How else would you create the logic in this class to alter the attribute? 
Also, careful with the `:permit` in the attr_reader, we didn't want to expose a method called `.permit` so we don't want it in the attr_reader.